api/internal/crawl/cmd/crawler/crawler.go

```diff
@@ -117,6 +143,13 @@ func main() {
 	}
 
 	crawlers := []crawler.Crawler{ghCrawler}
-	crawler.CrawlFromSeed(ctx, seedDocs, crawlers, docConverter, index, seen)
-	crawler.CrawlGithub(ctx, crawlers, docConverter, index, seen)
+
```

Comment:
would be able to swictch on CrawlMode here

Reply:
Done